orttraining/orttraining/core/optimizer/graph_transformer_utils.cc

```diff
@@ -52,11 +51,9 @@ namespace transformer_utils {
 
 std::vector<std::unique_ptr<GraphTransformer>> GeneratePreTrainingTransformers(
     TransformerLevel level,
-    std::unordered_set<std::string>& weights_to_train,
+    const std::unordered_set<std::string>& weights_to_train,
     const TrainingSession::TrainingConfiguration::GraphTransformerConfiguration& config,
     const IExecutionProvider& execution_provider,
-    std::unordered_map<std::string, std::string>& updated_weight_names,
-    std::unordered_map<std::string, TrainingSession::PartitionInfo>& weight_partition_info,
```

Comment:
merge error? 
I think there are still needed.

Reply:
I see... you moved megaton transformation out of graph transformation.

Reply:
Removed by intention. Because those two information are only needed by Megatron Transformer. Now we move that out of the transformer list got from this function.